markets/aave/commons.ts

```diff
@@ -58,8 +63,8 @@ export const CommonsConfig: ICommonConfiguration = {
     [eEthereumNetwork.rinkeby]: ZERO_ADDRESS,
   },
   FallbackOracle: {
-    [eEthereumNetwork.kovan]: "0x50913E8E1c650E790F8a1E741FF9B1B1bB251dfe",
-    [eEthereumNetwork.main]: "0x5b09e578cfeaa23f1b11127a658855434e4f3e09",
+    [eEthereumNetwork.kovan]: ZERO_ADDRESS,
+    [eEthereumNetwork.main]: ZERO_ADDRESS,
```

Comment:
should this be `ZERO_ADDRESS`?

Reply:
saw test for fallback to zero so i assume yes

Reply:
yes FallbackOracle is a deprecated part of the AaveOracle and should be ZERO_ADDRESS